### ISSUE TITLE
Python VEnv Built Error

### ISSUE THREAD

**Issue by AlexanderInUM** [veupnea/CCP#1]
I encountered the problem shown below when I was installing the `v_env_requirements`.
```
 × Preparing metadata (pyproject.toml) did not run successfully.
        │ exit code: 1
        ╰─> [275 lines of output]
            setup.py:67: RuntimeWarning: NumPy 1.19.3 may not yet support Python 3.10.
              warnings.warn(
            Running from numpy source directory.
            setup.py:480: UserWarning: Unrecognized setuptools command, proceeding with generating Cython sources and expanding templates
              run_build = parse_setuppy_commands()
            Processing numpy/random\_bounded_integers.pxd.in
            Processing numpy/random\bit_generator.pyx
            Processing numpy/random\mtrand.pyx
            Processing numpy/random\_bounded_integers.pyx.in
            Processing numpy/random\_common.pyx
            Processing numpy/random\_generator.pyx
            Processing numpy/random\_mt19937.pyx
            Processing numpy/random\_pcg64.pyx
            Processing numpy/random\_philox.pyx
            Processing numpy/random\_sfc64.pyx
            Cythonizing sources
            blas_opt_info:
            blas_mkl_info:
            No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
            customize MSVCCompiler
```

<br>

<br>

<br>

What is your Python version?

**Comment by apanay20** [comment]
Hello Alexander.

I tried to recreate the venv. I used Python 3.7.9 and run the steps written in the documentation.
The only addition is that after the installation of the packages from the .txt file, I had to use a specific version of "importlib-metadata" as mlagents package need that. (pip install importlib-metadata==4.4).

1. pip install virtualenv
2. python -m venv venv
3. venv\Scripts\activate
4. python -m pip install --upgrade pip
5. pip install -r venv_requirements.txt
6. pip install importlib-metadata==4.4
